tests/greenheart/test_openmdao.py

```diff
@@ -322,15 +322,15 @@ def test_greenheart_component(subtests):
 
     # TODO base this test value on something
     with subtests.test("lcoh"):
-        assert prob["lcoh"][0] == approx(3.04011846, rel=rtol)
+        assert prob["lcoh"][0] == approx(3.1691092704830357, rel=rtol)
```

Comment:
same comment here. We don't need to worry about variable sizing in the openmdao tests except perhaps in the optimization test to make sure the size is changing as we progress through the optimization.